HackYourFuture-CPH/FP-annotatetheweb-class10
# #169 LoginPage, RegisterPage, Home saves/uses user_id, screenshot_key to/from local storage

Changes in this PR:
- User_id and Screenshot_id are saved into local storage instead of context files
- Homepage: 
	- when user logs out, local storage user_id is set to 1, and screenshot_key is set to ""
	- screensot_key is saved into local storage
- LoginPage:
	- Added LoginPage css file from storybook, enabled images
	- New request for getting user_id by email
	- User_id is saved to local storage
- RegisterPage:
	- New request for getting user_id by email
- Router and controllers:
	- Created new route for getting user_id by email
- Database:
	- Created new column for email
- Added documentation in swagger

**How to test:**

- Open Google Devtools, go to Application, login or register and check if the user_id in local storage changes. 
- Log out (you can do that on homepage if you are already signed in), and chechk if user_id and screenshot_key (if applicable) turns to 1 . (screenshot_key should turn to ""). 
- Create a screenshot, and see if screenshot_key appears in local storage. 
- Try signing in/registering by social media logins. I noticed that when I use my email that has permission to the firebase project, there's no new user created in database.
